### PR TITLE
Fill with epm last commits 

### PR DESCRIPTION
**Pull request by VoR0220**
* Fill with missing last EPM commit per #900 and the consolidation checklist in the PR
* Requires #1108

### CODE REVIEW

File: pkgs/abi/core.go [hunk 1]
```diff
@@ -104,17 +104,17 @@ func packInterfaceValue(typ Type, val string) (interface{}, error) {
...
-			return hex.RightPadBytes(bytez.Bytes(), typ.SliceSize), nil
+			return common.RightPadBytes(bytez.Bytes(), typ.SliceSize), nil
 		} else if typ.Elem.T == BytesTy || typ.Elem.T == FixedBytesTy {
 			val = strings.Trim(val, "[]")
 			arr := strings.Split(val, ",")
 			var sliceOfFixedBytes [][]byte
 			for _, str := range arr {
 				bytez := bytes.NewBufferString(str)
-				sliceOfFixedBytes = append(sliceOfFixedBytes, hex.RightPadBytes(bytez.Bytes(), 32))
+				sliceOfFixedBytes = append(sliceOfFixedBytes, common.RightPadBytes(bytez.Bytes(), 32))
```
Comment (zramsay) [inline pkgs/abi/core.go]: common is deprecated except for docs. it's also not even included in the import paths

File: pkgs/abi/core.go [hunk 2]
```diff
@@ -420,14 +422,14 @@ func getStringValue(value interface{}, output Argument) (string, error) {
 			case 8, 16, 32, 64:
 				return fmt.Sprintf("%v", value), nil
 			default:
-				return hex.S256(value.(*big.Int)).String(), nil
+				return common.S256(value.(*big.Int)).String(), nil
```
Comment (zramsay) [inline pkgs/abi/core.go]: gotta bring this is in neatly. the hex package was my temp hack to remove from common permanently

File: pkgs/abi/core.go [hunk 3]
```diff
@@ -420,14 +422,14 @@ func getStringValue(value interface{}, output Argument) (string, error) {
...
 			}
 		case UintTy:
 			switch typ.Size {
 			case 8, 16, 32, 64:
 				return fmt.Sprintf("%v", value), nil
 			default:
-				return hex.U256(value.(*big.Int)).String(), nil
+				return common.U256(value.(*big.Int)).String(), nil
```
Comment (zramsay) [inline pkgs/abi/core.go]: ibid

File: pkgs/abi/core.go (hunk 1 above)
Comment (VoR0220) [inline pkgs/abi/core.go]: shit I forgot about that. Copy pasted. My bad.

File: pkgs/abi/core.go (hunk 3 above)
Comment (VoR0220) [inline pkgs/abi/core.go]: these get fixed in https://github.com/eris-ltd/eris-cli/pull/1113. Should I just combine the two PRs?

File: pkgs/abi/core.go (hunk 2 above)
Comment (zramsay) [inline pkgs/abi/core.go]: intepreter

File: pkgs/abi/core.go (hunk 3 above)
Comment (zramsay) [inline pkgs/abi/core.go]: nope. fix the conflict on this branch

**Comment by VoR0220** [comment]
@zramsay fixed.

**Pull request merged**